model/facade/action/GetWeeklyGoalSummaryAction.php

```diff
@@ -106,14 +106,4 @@ protected function doExecute() {
 
 	}
 
-}
-
-/*//Test code
-
-$user = new UserVO();
-$user->setLogin('jaragunde');
-$action = new GetPersonalSummaryByUserIdDateAction($user, date_create('2009-12-01'));
-//var_dump($action);
-$result = $action->execute();
-var_dump($result);
-*/
+}
```

Comment:
This is the kind of trivial change I would have amended into the previous commit O:-)

No need to do it now, just take it into account in the future.